src/site/zh/xdoc/java-api.xml

```diff
@@ -65,10 +65,10 @@
   <p>这部分内容剩余的示例将假设你使用了这种目录结构。</p>
   </subsection>
 
-  <subsection name="SqlSessions" id="sqlSessions">
-  <p>使用 MyBatis 的主要 Java 接口就是 SqlSession。你可以通过这个接口来执行命令，获取映射器和管理事务。我们会概括讨论一下 SqlSession 本身，但是首先我们还是要了解如何获取一个 SqlSession 实例。SqlSessions 是由 SqlSessionFactory 实例创建的。SqlSessionFactory 对象包含创建 SqlSession 实例的所有方法。而 SqlSessionFactory 本身是由 SqlSessionFactoryBuilder 创建的，它可以从 XML、注解或手动配置 Java 代码来创建 SqlSessionFactory。</p>
+  <subsection name="SqlSession" id="sqlSession">
```

Comment:
`id` is used as an [anchor](https://mybatis.org/mybatis-3/java-api.html#sqlSessions), so it may be better to keep it the same as the English doc.

```suggestion
  <subsection name="SqlSession" id="sqlSessions">
```